packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -430,8 +431,21 @@ impl VTag {
             .insert(key, value.into_prop_value());
     }
 
+    /// Add event listener on the [VTag]'s  [Element]
+    pub fn add_listener(&mut self, listener: Rc<dyn Listener>) -> bool {
+        if let Listeners::Pending(listeners) = &mut self.listeners {
+            let mut listeners = mem::take(listeners).into_vec();
+            listeners.push(Some(listener));
+
+            self.set_listeners(listeners.into_boxed_slice());
+            true
+        } else {
+            false
```

Comment:
What does the bool mean? False means it failed to add listener? Could you specify that in the code comment above?

Reply:
Ok, I added a comment.